packer/gcp-packer.json.pkr.hcl

```diff
@@ -15,25 +15,31 @@
 # views them; you can change their type later on. Read the variables type
 # constraints documentation
 # https://www.packer.io/docs/from-1.5/variables#type-constraints for more info.
+
 variable "account_file" {
   type    = string
   default = ""
 }
 
 variable "project_id" {
-  type = string
-  default = ""
+  type        = string
+  default     = ""
   description = "The project ID that will be used to launch instances and store images."
 }
 
 variable "zone" {
-  type = string
-  default = "europe-west1-b"
+  type        = string
+  default     = "europe-west1-b"
   description = "The zone in which to launch the instance used to create the image."
 }
 
-# "timestamp" template function replacement
-locals { timestamp = regex_replace(timestamp(), "[- TZ:]", "") }
+variable "tag" {
+  type        = string
+  default     = ""
+  description = "The image tag"
+}
+
+locals { tag = substr("${var.tag}", 0, 7) }
```

Comment:
Our tags would comply with Semver (e.g. 1.2.3) so we shouldn't enforce taking a substring out of it